server/src/main/java/org/opensearch/telemetry/tracing/TracerSettings.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry.tracing;
+
+import org.opensearch.common.settings.ClusterSettings;
+import org.opensearch.common.settings.Setting;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.common.unit.TimeValue;
+
+/**
+ * Wrapper class to encapsulate tracing related settings
+ */
+public class TracerSettings {
+    public static final Setting<Integer> TRACER_EXPORTER_BATCH_SIZE_SETTING = Setting.intSetting(
+        "tracer.exporter.batch_size",
+        512,
+        1,
+        Setting.Property.NodeScope,
+        Setting.Property.Dynamic
+    );
+    public static final Setting<Integer> TRACER_EXPORTER_MAX_QUEUE_SIZE_SETTING = Setting.intSetting(
+        "tracer.exporter.max_queue_size",
+        2048,
+        1,
+        Setting.Property.NodeScope,
+        Setting.Property.Dynamic
+    );
+    public static final Setting<TimeValue> TRACER_EXPORTER_DELAY_SETTING = Setting.timeSetting(
+        "tracer.exporter.delay",
+        TimeValue.timeValueSeconds(2),
+        Setting.Property.NodeScope,
+        Setting.Property.Dynamic
+    );
+    public static final Setting<Level> TRACER_LEVEL_SETTING = new Setting<>(
+        "tracer.level",
+        Level.ROOT.name(),
+        Level::fromString,
+        Setting.Property.NodeScope,
+        Setting.Property.Dynamic
+    );
+
+    public static final Setting<Boolean> TRACER_ENABLED_SETTING = Setting.boolSetting(
```

Comment:
This setting is applicable to all traces, the other settings (TRACER_EXPORTER_XXX) are very specific to Otel, we should move it to Otel plugin I think

Reply:
> The exporter though looks like Otel specific setting but I think we would have exporter concept in other telemetries too. 

That is totally fine, the other plugins would introduce settings related to their exporters, some may overlap, some not

Reply:
Done